### PR TITLE
Create d411d3c1-28eb-458c-b83a-70bfd4aa95a0.yml

### PR DESCRIPTION
**Pull request by 0xGotcha**
UEFI Modified Windows Boot Loader

### CODE REVIEW

**Review by 0xGotcha — COMMENTED** [review]
Changed T&T to better fit the TTP. 

Adversaries may abuse Pre-OS Boot mechanisms as a way to establish persistence on a system. During the booting process of a computer, firmware and various startup services are loaded before the operating system. These programs control flow of execution before the operating system takes control.[1]

**Review by 0xGotcha — COMMENTED** [review]
removed tags line

**Comment by khyberspache** [comment]
Assigning myself, will review tomorrow AM.

File: persistence/d411d3c1-28eb-458c-b83a-70bfd4aa95a0.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  id: T1542
+  name: Pre-OS Boot
+platforms:
+  windows:
+    cmd:
+      command: 'mountvol x: /s'
+metadata:
+  authors:
+    - Logical
+  version: 1
```
Comment (khyberspache) [inline persistence/d411d3c1-28eb-458c-b83a-70bfd4aa95a0.yml:15]: add `tags: []`

File: persistence/d411d3c1-28eb-458c-b83a-70bfd4aa95a0.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+id: d411d3c1-28eb-458c-b83a-70bfd4aa95a0
+name: UEFI Modified Windows Boot Loader
```
Comment (khyberspache) [inline persistence/d411d3c1-28eb-458c-b83a-70bfd4aa95a0.yml:2]: Mount hidden EFI partition?

**Review by khyberspache — CHANGES_REQUESTED** [review]
small changes, I like this!

File: persistence/d411d3c1-28eb-458c-b83a-70bfd4aa95a0.yml (hunk 2 above)
Comment (0xGotcha) [reply persistence/d411d3c1-28eb-458c-b83a-70bfd4aa95a0.yml:2]: Technically you can remove it back after you mount yes.

**Pull request closed without merging**